openwrt/docker/Dockerfile

```diff
@@ -12,6 +12,8 @@ RUN apk add --no-cache bash \
     py3-click \
     nano \
     vim \
+    tcpdump \
+    busybox-extras \
```

Comment:
Hi @Takalele 
we are transitioning to a base image, like here https://github.com/hellt/vrnetlab/blob/master/sros/docker/Dockerfile

with the latest base image being 0.2.1

It does not have busybox-extras, so you can add it in a `RUN` instruction after FROM